src/afrey/boot_asset_fingerprint.clj

```diff
@@ -27,8 +36,9 @@
                           (core/output-files)
                           (core/by-ext (or extensions [".html"])))
             file-hashes (into {}
-                          (map (juxt :path :hash))
-                          (core/output-files fileset))]
+                              (map (juxt :path :hash))
+                              (core/by-ext (or rename-extensions [".js" ".css"]) (core/output-files fileset)))
```

Comment:
I like that idea, 
I was thinking something like:

1. Evaluating the files captured by `extensions`
2. Collect file paths
3. Filter files from the fileset and fingerprint
4. Replace template values with fingerprinted version

Does that sounds like a reasonable approach? @RadicalZephyr